cmd/crc/cmd/root.go

```diff
@@ -55,6 +57,8 @@ func init() {
 		logging.Fatal(err.Error())
 	}
 
+	secretConfig = newSecretConfig()
```

Comment:
looking again at this today, this can be improved, we don't need to expose the storage side of things, we can instead extend the `Config` objects `Set`, `Get`, and `Unset` methods to take care of an additional `Secret` config type like strings, bools etc. and hide the implementation detail of the secret config store, will rework this part of the PR again.

cc @cfergeau